### PR TITLE
adding more comments to ViewController

### PR DESCRIPTION
**Pull request by elialfordjones**

### CODE REVIEW

File: EliTweets/ViewController.swift [hunk 1]
```diff
@@ -4,7 +4,8 @@
 //
 //  Created by Eli Alford-Jones on 6/20/15.
 //  Copyright (c) 2015 Eli Alford-Jones. All rights reserved.
-//  test commit comment for tanner
+//  test commit comment for tanner more comments
+//  moore
```
Comment (tannerwelsh) [inline EliTweets/ViewController.swift:8]: Nice job.

Comment (tannerwelsh) [inline EliTweets/ViewController.swift:8]: :dancer:

Comment (elialfordjones) [inline EliTweets/ViewController.swift:8]: misspelled word

File: EliTweets/ViewController.swift [hunk 2]
```diff
@@ -4,7 +4,8 @@
 //
 //  Created by Eli Alford-Jones on 6/20/15.
 //  Copyright (c) 2015 Eli Alford-Jones. All rights reserved.
-//  test commit comment for tanner
```
Comment (tannerwelsh) [inline EliTweets/ViewController.swift:7]: thanks!

**Pull request closed without merging**